README.md

```diff
@@ -0,0 +1,78 @@
+# Node.js-call-CRUD
+[![pipeline status][pipeline]][git-link]
+[![coverage report][coverage]][git-link]
+
+## Getting Started
+
+### Summary
+%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%
+
+This is an Example of a simple Node application calling the CRUD service.
+It shows how to access the riders collection using pur custom plugin for the requests.
+The CRUD path to access is taken from the env variable CRUD_PATH.
+
+For using the example provided you can run:
+
+```shell
+npm run start:local
+```
+
+This command will launch the service on `localhost:3000` with the environment variables defined
+in this [file](./default.env).
+
+### Local Development
+To develop the service locally you need:
+- Node 10+
+- fastify 2.10.0+
+- @mia-platform/custom-plugin-lib 1.0.5+
```

Comment:
The versions of dependencies are written in `package.json` and `package-lock.json`. We should not write in README, or it will be difficult to keep this up-to-date.